azurelinuxagent/common/cgroupapi.py

```diff
@@ -160,6 +164,14 @@ def _foreach_legacy_cgroup(operation):
                 shutil.rmtree(cgroup, ignore_errors=True)
         return len(legacy_cgroups)
 
+    @staticmethod
```

Comment:
let's move these 2 to SystemdCgroupsApi, since slices are systemd concepts

and let's remove them from FileSystemCgroupsApi